analysis/src/CompletionBackEnd.ml

```diff
@@ -931,6 +931,30 @@ let findLocalCompletionsForValuesAndConstructors ~(localTables : LocalTables.t)
        (processLocalModule ~prefix ~exact ~env ~localTables);
   List.rev_append localTables.resultRev valuesFromOpens
 
+let findLocalCompletionsForValues ~(localTables : LocalTables.t) ~env ~prefix
```

Comment:
Slimmed down version of a pre-existing function (but that handles both this + constructors). We don't want constructors in typed completion, since they are handled by the typed completion itself.